Library/Homebrew/sbom.rb

```diff
@@ -0,0 +1,320 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cxxstdlib"
+require "options"
+require "json"
+require "development_tools"
+require "extend/cachable"
+
+# Rather than calling `new` directly, use one of the class methods like {SBOM.create}.
+class SBOM
```

Comment:
better call it SBOM_SPDX - since it is pretty much SPDX specific.

Reply:
I think we can do that if we ever decide to support alternative outputs.